layouts/index.html

```diff
@@ -0,0 +1,114 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>NicksPatties: A Nick Patti Portfolio</title>
+    <link href="fontawesome/css/fontawesome.css" rel="stylesheet">
+    <link href="fontawesome/css/brands.css" rel="stylesheet">
+    <link href="fontawesome/css/solid.css" rel="stylesheet">
```

Comment:
Don't need this since I only use the brands.css file. Also can just use the minified versions of those files.